### PR TITLE
Update akka projection to 1.2.0

### PR DESCRIPTION
**Pull request by Tochemey**
from 1.10 to 1.2.0

https://github.com/akka/akka-projection/releases/tag/v1.2.0

This PR is just to update the projection version which includes a new table. It has not breaking change effect.

### CODE REVIEW

File: code/migration/src/main/scala/com/namely/chiefofstate/migration/SchemasUtil.scala [hunk 1]
```diff
@@ -96,7 +96,8 @@ object SchemasUtil {
    * @return the DBIOAction creating the table and offset
    */
   private[migration] def createReadSideOffsetsStmt(
-      tableName: String = "read_side_offsets"): DBIOAction[Unit, NoStream, Effect] = {
+      tableName: String = "read_side_offsets",
+      projectionsTableName: String = "projections"): DBIOAction[Unit, NoStream, Effect] = {
```
Comment (zenyui) [inline code/migration/src/main/scala/com/namely/chiefofstate/migration/SchemasUtil.scala:100]:
```suggestion
      projectionsTableName: String = "read_sides"): DBIOAction[Unit, NoStream, Effect] = {
```

File: code/service/src/main/resources/application.conf [hunk 2]
```diff
@@ -138,6 +138,8 @@ akka {
 				schema = ${jdbc-default.schema}
 				# the database table name for the offset store
 				table = "read_side_offsets"
+				# the database table name for the projection management
+				management-table = "projections"
```
Comment (zenyui) [inline code/service/src/main/resources/application.conf:142]:
```suggestion
				management-table = "read_sides"
```

**Review by zenyui — APPROVED** [review]
commit them i'll approve again

File: code/migration/src/test/scala/com/namely/chiefofstate/migration/versions/v6/V6Spec.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  ".snapshot" should {
+    "create the new journal, snapshot and read side stores" in {
+      val version = V6(journalJdbcConfig)
+      Await.result(journalJdbcConfig.db.run(version.snapshot()), Duration.Inf) shouldBe {}
+      DbUtil.tableExists(journalJdbcConfig, "event_journal") shouldBe true
+      DbUtil.tableExists(journalJdbcConfig, "event_tag") shouldBe true
+      DbUtil.tableExists(journalJdbcConfig, "state_snapshot") shouldBe true
+      DbUtil.tableExists(journalJdbcConfig, "read_side_offsets") shouldBe true
+      DbUtil.tableExists(journalJdbcConfig, "projections") shouldBe true
```
Comment (Tochemey) [inline code/migration/src/test/scala/com/namely/chiefofstate/migration/versions/v6/V6Spec.scala:44]:
```suggestion
      DbUtil.tableExists(journalJdbcConfig, "read_sides") shouldBe true
```

**Review by zenyui — APPROVED** [review]

**Pull request merged**